releases/fluentd-elasticsearch-aws.yaml

```diff
@@ -0,0 +1,67 @@
+repositories:
+  # Cloud Posse incubator repo of helm charts
+  - name: "cloudposse-incubator"
+    url: "https://charts.cloudposse.com/incubator/"
+
+releases:
+
+  #######################################################################################
+  ## Fluentd to AWS Elasticsearch                                                      ##
+  ## Forward logs to AWS Elasticsearch with fluentd                                    ##
+  #######################################################################################
+
+  #
+  # References:
+  #   - https://github.com/cloudposse/charts/blob/master/incubator/fluentd-kubernetes-aws/values.yaml
+  #
+
+  - name: "fluentd-elasticsearch-aws"
+    namespace: "monitoring"
+    labels:
+      chart: "fluentd-kubernetes-aws"
+      repo: "cloudposse-incubator"
+      component: "fluentd"
+      namespace: "monitoring"
+      default: "false"
+    chart: "cloudposse-incubator/fluentd-kubernetes-aws"
+    version: "0.1.0"
+    wait: true
+    installed: {{ env "FLUENTD_ELASTICSEARCH_AWS_INSTALLED" | default "true" }}
+    values:
+      - nameOverride: "fluentd-elasticsearch-aws"
+        image:
+          repository: '{{ env "FLUENTD_ELASTICSEARCH_IMAGE_REPO" | default "fluent/fluentd-kubernetes-daemonset" }}'
+          tag: '{{ env "FLUENTD_ELASTICSEARCH_IMAGE_TAG" | default "v1.3.3-debian-elasticsearch-1.8" }}'
+          pullPolicy: '{{ env "FLUENTD_ELASTICSEARCH_IMAGE_PULL_POLICY" | default "IfNotPresent" }}'
+        # Leave role empty if you are not uisng IAM based access control
+        role: '{{ env "FLUENTED_ELASTICSEARCH_IAM_ROLE" | default "" }}'
+        elasticsearch:
+          endpoint: '{{ requiredEnv "FLUENTD_ELASTICSEARCH_ENDPOINT" }}'
+          region: '{{ coalesce (env "FLUENTD_ELASTICSEARCH_REGION") (env "AWS_REGION") (env "AWS_DEFAULT_REGION") }}'
+
+        prometheus:
+          createService: {{ env "FLUENTD_PROMETHEUS_CREATE_SERVICE" | default "true" }}
+          createServiceMonitor: {{ env "FLUENTD_PROMETHEUS_CREATE_SERVICE_MONITOR" | default "true" }}
+          createRule: {{ env "FLUENTD_PROMETHEUS_CREATE_RULE" | default "true" }}
+          labels:
+            app: prometheus-operator
+            release: prometheus-operator
+        env:
+          FLUENT_ELASTICSEARCH_BUFFER_CHUNK_LIMIT_SIZE: '{{- env "FLUENTD_ELASTICSEARCH_BUFFER_CHUNK_LIMIT_SIZE" | default "8M" }}'
```

Comment:
Nit: Seems like `cloudposse-incubator/fluentd-kubernetes-aws` should try to use `FLUENTD_` prefix

Reply:
The `FLUENT_*` environment variables are defined in/by the `fluent/fluentd-kubernetes-daemonset` Docker image, not the chart.